### ISSUE TITLE
node Process after shutdown app debug

### ISSUE THREAD

**Issue by hartov** [joshberry/dotnetcore-angular-ssr#3]
Lets check ng services from system monitoring.
after every launch by "F5" you could see new process "ng" wich reserve ~500mb of RAM memory

**Comment by joshberry** [comment]
Good point! I'm not clear on what you're suggesting as a solution. Can you elaborate a little on how to fix? 

This appears to be a problem with the official dotnet core angular template `dotnet new angular` as well. I'm seeing stranded node processes after every debug session.

**Comment by hartov** [comment]
+ Code Helper Issure + ~200mb
